documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -0,0 +1,63 @@
+
+[id='deploying-the-topic-operator-standalone']
+= Deploying the Topic Operator standalone
+
+Deploying the Topic Operator as a standalone component is more complicated than installing it using the Cluster Operator, but is more flexible.
+For instance is can operate _with_ any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+.Prerequisites
+
+* An existing Kafka cluster for the Topic Operator to connect to.
+
+.Procedure
+
+. Edit the `examples/install/cluster-operator/05-Deployment-strimzi-topic-operator.yaml` resource. You will need to change the following
```

Comment:
Will the file name always include 'strimzi'?

Reply:
@scholzj ^^

Reply:
Yes, I think so. It would be a lot of work to change that (and test) for very little benefit.